### DISCUSSION TITLE
Display does not work

### DISCUSSION THREAD

**Discussion by Denis-AAA** [Desuuuu/klipper#152] [Q&A]
display: DWIN DMT 48270c043_06WT connected to USART 1.
board: BTT Octopus Pro.
Installed Klipper  from here (dgus-reloaded-230526-0): https://github.com/Desuuuu/klipper
Firmware display from here (V 1.0.2) : https://github.com/Desuuuu/DGUS-reloaded-Klipper/releases
file T5UID1.CFG from here: https://github.com/Desuuuu/DGUS-reloaded-Klipper/tree/master/extras/flash_reset
Added to printer.cfg:
[t5uid1]
firmware: dgus_reloaded
machine_name: Ender 5 Plus
volume: 10
brightness: 50
z_min: 0
z_max: 410

However the display does not work. Only shows the Klipper logo. I don't understand what I'm doing wrong?
Help me please

**Comment by Desuuuu** [comment]
Sounds like a communication issue between the board and the touchscreen. I suggest checking that your wiring is correct (TX mainboard <-> RX touchscreen | RX mainboard <-> TX touchscreen). Also make sure you're using the correct RX/TX pins on the touchscreen (the inner 2 pins IIRC).
